lnst/RecipeCommon/Perf/Measurements/Results/RDMABandwidthMeasurementResults.py

```diff
@@ -0,0 +1,31 @@
+from lnst.RecipeCommon.Perf.Measurements.BaseMeasurement import BaseMeasurement
+from lnst.RecipeCommon.Perf.Measurements.Results.BaseMeasurementResults import BaseMeasurementResults
+from lnst.RecipeCommon.Perf.Results import PerfResult
+
+
+class RDMABandwidthMeasurementResults(BaseMeasurementResults):
+    def __init__(self, measurement: BaseMeasurement, flow, bandwidth: PerfResult):
```

Comment:
Yes, because I couldn't import it because of a cyclic dependency. I could work around this the intended way by doing the following:
```
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    # import flow
```
But I feel like this would be pointless, because we currently don't do type-checking. Should I include it?

Reply:
i think i'd prefer at least to use the string version: `flow: "Flow"` just so we're somewhat consistent within the function parameters.

Reply:
Fixed, please resolve.